etna/settings.py

```diff
@@ -38,6 +38,21 @@ def _is_torch_available():
         return False
 
 
+def _is_chronos_available():
+    true_case = (
+        _module_available("accelerate")
+        & _module_available("safetensors")
```

Comment:
We haven't such dependency in `pyproject.toml`.

Reply:
Do we need to put `safetensors` or `tokenizers` in `pyproject.toml` if it is dependency of `transformers` or `accelerate`?

Reply:
I think we should place here only dependencies that we have in `pyproject.toml` in corresponding extras.